File: version_3/code-samples/ClientApi/Session/SavingChanges.cs
using Raven.Client.Document;
using Raven.Documentation.CodeSamples.Orders;

namespace Raven.Documentation.CodeSamples.ClientApi.Session
{
	public class SavingChanges
	{
		private interface IInterface
		{
			#region saving_changes_1
			void SaveChanges();
			#endregion
		}

		public SavingChanges()
		{
			using (var store = new DocumentStore())
			{
				using (var session = store.OpenSession())
				{
					#region saving_changes_2
					// storing new entity
					session.Store(new Employee
						              {
							              FirstName = "John", 
										  LastName = "Doe"
						              });

					session.SaveChanges();
					#endregion
				}
			}
		}
	}
}

File: Documentation/3.0/Samples/java/src/test/java/net/ravendb/clientapi/session/SavingChanges.java
package net.ravendb.clientapi.session;

import net.ravendb.client.IDocumentSession;
import net.ravendb.client.IDocumentStore;
import net.ravendb.client.document.DocumentStore;
import net.ravendb.samples.northwind.Employee;


public class SavingChanges {

  @SuppressWarnings("unused")
  private interface IInterface {
    //region saving_changes_1
    public void saveChanges();
    //endregion
  }

  public SavingChanges() throws Exception {
    try (IDocumentStore store = new DocumentStore()) {
      try (IDocumentSession session = store.openSession()) {
        //region saving_changes_2
        // storing new entity
        Employee employee = new Employee();
        employee.setFirstName("John");
        employee.setLastName("Doe");
        session.store(employee);

        session.saveChanges();
        //endregion
      }
    }
  }
}
